samples/sensor/sm351lt/sample.yaml

```diff
@@ -0,0 +1,7 @@
+sample:
+  name: SM351LT Magnetoresistive Sensor Example
+tests:
+  sample.sensor.sm351lt:
+    harness: sensor
+    tags: samples sensor
+    depends_on: gpio sm351lt
```

Comment:
Does the bt510 board (or any board) have `sm3511t` in its supported list? If not, this sample will not get built in CI

Reply:
The bt510 board is pending review here: https://github.com/zephyrproject-rtos/zephyr/pull/27400
But the dts doesn't have the sm351lt in it at present, the idea was to get the basic board added alongside this, then get the bt510 board updated to include this otherwise I'm assuming the board would fail submission because the automated checks would fail and say sm351lt not a known driver, or can we add it prior to the driver existing?